Resources/Markdown/docs/builds.md

```diff
@@ -0,0 +1,53 @@
+---
+page-title: Builds FAQ
+description: Frequently Asked Questions about the Build System
+---
+
+## The Swift Package Index Build System
+
+* [What is the Build System?](#build-system)
+* [Why does a package show missing or incomplete compatibility?](#no-builds)
+* [What revision is the default branch built for?](#what-revision)
+* [How are packages built?](#built-how)
+* [When was a package last built?](#last-built)
+* [Is it possible to hide failing builds for unsupported platforms?](#hide-failing-builds)
+* [If a build is showing failed incorrectly, how can I fix it?](#fix-false-negative)
+* [If a build is showing an error that seems unrelated to the build, how can I fix it?](#unrelated-error)
+
+<h3 id="build-system">What is the Build System?</h3>
+
+A Swift package manifest includes support for declaring [Swift version support](https://developer.apple.com/documentation/swift_packages/package/3197887-swiftlanguageversions) and [platform support](https://developer.apple.com/documentation/swift_packages/package/3197886-platforms). There are a few problems with relying on this data, so the Swift Package Index takes a different approach. What better way to determine compatibility with a specific version of Swift on a particular platform than to build it?
+
+The Swift Package Index build system is a mechanism that keeps a package page always up to date with real-world build results targeting various Swift versions across multiple platforms, including Linux.
+
+<h3 id="no-builds">Why does a package show missing or incomplete compatibility?</h3>
+
+The Swift Package Index will pick up new default branch revisions and releases automatically. It can take a little while for build results to become available after a commit or new version release though, so if the compatibility matrix shows a clock symbol then builds are still pending, and results are on their way!
+
+<h3 id="what-revision">What revision is the default branch built for?</h3>
+
+The Swift Package Index does not currently display which revision the default branch tracks. If a default branch revision changed more than a few hours ago, everything should be up to date. If there looks to be a problem with a build not updating after several hours, please [raise an issue](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/new).
```

Comment:
Just as a reminder and for discussing - perhaps we should set up an issue template specifically around build issues/requests.